build-ps/percona-server-8.0_builder.sh

```diff
@@ -281,18 +284,48 @@ get_sources(){
     sed -i "s:@@REVISION@@:${REVISION}:g" build-ps/percona-server.spec
     sed -i "s:@@RPM_RELEASE@@:${RPM_RELEASE}:g" build-ps/percona-server.spec
     sed -i "s:@@BOOST_PACKAGE_NAME@@:${BOOST_PACKAGE_NAME}:g" build-ps/percona-server.spec
+
     cd ${WORKDIR}/percona-server
     tar --owner=0 --group=0 --exclude=.bzr --exclude=.git -czf ${PSDIR}.tar.gz ${PSDIR}
 
     mkdir $WORKDIR/source_tarball
     mkdir $CURDIR/source_tarball
     cp ${PSDIR}.tar.gz $WORKDIR/source_tarball
     cp ${PSDIR}.tar.gz $CURDIR/source_tarball
+    cp percona-server-8.0.properties $WORKDIR/source_tarball
+    cp percona-server-8.0.properties $CURDIR/source_tarball
     cd $CURDIR
     rm -rf percona-server
     return
 }
 
+enable_zenfs() {
+    local mode=$1
+
+    cp $CURDIR/source_tarball/percona-server-8.0.properties $WORKDIR
+    source $WORKDIR/percona-server-8.0.properties
+
+    rm -rf storage/rocksdb/rocksdb_plugins/zenfs
+    git clone https://github.com/westerndigitalcorporation/zenfs.git storage/rocksdb/rocksdb_plugins/zenfs
+    if [[ $mode == "tarball" ]]; then
+        rm build-ps/build-binary.sh
+        curl https://raw.githubusercontent.com/Sudokamikaze/percona-server/PS-7764/build-ps/build-binary.sh --output build-ps/build-binary.sh
+        chmod +x build-ps/build-binary.sh
+        git clone --recursive https://github.com/percona-ysorokin/rocksdb.git -b percona_wdc $WORKDIR/TARGET/rocksdb-source
+    elif [[ $mode == "debian" ]]; then
+        mkdir builddir
+        curl https://raw.githubusercontent.com/Sudokamikaze/percona-server/PS-7764/build-ps/debian/rules_zenfs --output debian/rules_zenfs
```

Comment:
why not `cp rules.zenfs rules` ?

Reply:
Because this file is not present in Yura's repository